2022-11-28 21:00:59-DEBUG-logging_using_confg.py-10-This is root Logger, debug
2022-11-28 21:00:59-ERROR-logging_using_confg.py-18-invalid literal for int() with base 10: 'a'
Traceback (most recent call last):
  File "D:/PycharmProjects/common_modules_study/logging_module/logging_using_confg.py", line 16, in <module>
    int('a')
ValueError: invalid literal for int() with base 10: 'a'
2022-11-28 21:02:02-DEBUG-logging_using_confg.py-14-This is root Logger, debug
2022-11-28 21:02:02-ERROR-logging_using_confg.py-20-invalid literal for int() with base 10: 'a'
Traceback (most recent call last):
  File "D:/PycharmProjects/common_modules_study/logging_module/logging_using_confg.py", line 18, in <module>
    int('a')
ValueError: invalid literal for int() with base 10: 'a'
2022-11-28 21:02:29-DEBUG-logging_using_confg.py-14-This is Logger, debug
2022-11-28 21:02:29-ERROR-logging_using_confg.py-20-invalid literal for int() with base 10: 'a'
Traceback (most recent call last):
  File "D:/PycharmProjects/common_modules_study/logging_module/logging_using_confg.py", line 18, in <module>
    int('a')
ValueError: invalid literal for int() with base 10: 'a'
2022-11-28 21:02:39-DEBUG-logging_using_confg.py-11-This is Logger, debug
2022-11-28 21:02:39-ERROR-logging_using_confg.py-19-invalid literal for int() with base 10: 'a'
Traceback (most recent call last):
  File "D:/PycharmProjects/common_modules_study/logging_module/logging_using_confg.py", line 17, in <module>
    int('a')
ValueError: invalid literal for int() with base 10: 'a'
2022-11-28 21:20:05-DEBUG-logging_using_confg.py-11-This is Logger, debug
2022-11-28 21:20:05-ERROR-logging_using_confg.py-19-invalid literal for int() with base 10: 'a'
Traceback (most recent call last):
  File "D:/PycharmProjects/common_modules_study/logging_module/logging_using_confg.py", line 17, in <module>
    int('a')
ValueError: invalid literal for int() with base 10: 'a'
2022-11-28 21:20:56-DEBUG-logging_using_confg.py-10-This is Logger, debug
2022-11-28 21:20:56-ERROR-logging_using_confg.py-18-invalid literal for int() with base 10: 'a'
Traceback (most recent call last):
  File "D:/PycharmProjects/common_modules_study/logging_module/logging_using_confg.py", line 16, in <module>
    int('a')
ValueError: invalid literal for int() with base 10: 'a'
2022-11-28 21:21:10-DEBUG-logging_using_confg.py-10-This is Logger, debug
2022-11-28 21:21:10-ERROR-logging_using_confg.py-18-invalid literal for int() with base 10: 'a'
Traceback (most recent call last):
  File "D:/PycharmProjects/common_modules_study/logging_module/logging_using_confg.py", line 16, in <module>
    int('a')
ValueError: invalid literal for int() with base 10: 'a'
